tensorflow/compiler/xla/service/gpu/gpu_fusible.h

```diff
@@ -56,6 +69,10 @@ bool IsInputFusibleReduction(const HloInstruction& instr);
 bool ShapesCompatibleForMultiOutputFusion(const HloInstruction& instr1,
                                           const HloInstruction& instr2);
 
+// Whether `instr` can be the root of sibling multi-output fusible or
+// a suitable consumer for multi-output fusion
```

Comment:
Let's rephrase this a bit.
Whether `instr` is a candidate for sibling fusion or as consumer in a producer-consumer multi-output fusion. Note that sibling fusions have multiple outputs be definition.